_posts/0000-01-02-declanbond.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
The forms of thought are, in the first instance, displayed and stored as human language.